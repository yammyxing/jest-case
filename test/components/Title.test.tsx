import React from 'react';
import { render } from '@testing-library/react'
import Title from '@/components/Title';


describe('Title component', () => {
    it('should render large title', () => {
        // const { baseElement } = render(<Title title="hello large" type="small" />)
        // expect(baseElement).toMatchSnapshot();
        const { getByText } = render(<Title title="hello large" type="small" />)
        const content = getByText('hello large')
        expect(content).toMatchSnapshot();
    })

    it('should render small title', () => {
        // const { baseElement } = render(<Title title="hello small" type="large" />)
        // expect(baseElement).toMatchSnapshot();
        const { getByText } = render(<Title title="hello small" type="large" />)
        const content = getByText('hello small')
        expect(content).toMatchSnapshot();
    })
})