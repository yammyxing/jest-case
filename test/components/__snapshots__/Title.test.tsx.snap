// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title component should render large title 1`] = `
<div>
  hello large
</div>
`;

exports[`Title component should render small title 1`] = `
<div>
  hello small
</div>
`;
